packages/eslint-plugin-turbo/__tests__/lib/no-undeclared-env-vars.test.ts

```diff
@@ -537,3 +541,534 @@ ruleTester.run(RULES.noUndeclaredEnvVars, rule, {
     },
   ],
 });
+
+moduleRuleTester.run(RULES.noUndeclaredEnvVars, rule, {
```

Comment:
I've just copied all of the existing test cases, replacing `process.env` with `import.meta.env`. Let me know if there's a different way you'd prefer I do this.